js/parts/Dynamics.js

```diff
@@ -924,6 +924,7 @@ extend(Series.prototype, /** @lends Series.prototype */ {
         if (redraw) {
             chart.redraw(animation); // Animation is set anyway on redraw, #5665
         }
+        return point;
```

Comment:
We also need to update the doclet so that JSDoc is aware of the correct return value.